contrib/src/test/java/com/datatorrent/contrib/kafka/KafkaInputOperatorTest.java

```diff
@@ -234,7 +236,20 @@ public void testKafkaInputOperator_Invalid() throws Exception
   public void afterTest()
   {
     tupleCount.set(0);
```

Comment:
I think afterTest method will always be run even if there is an exception in before and test